uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestActivity.kt

```diff
@@ -324,7 +295,7 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     }
 
     override fun showPrebookConfirmationDialog(quoteType: QuoteType?, tripInfo: TripInfo) {
-        val activity = context as Activity
+        val activity = this as Activity
         val activityWasStartedForResult = activity.callingActivity != null
```

Comment:
after removing the one from the last message you can refactor this to `this.callingActivity`

Reply:
Done